src/funcx_common/task_storage/s3.py

```diff
@@ -38,9 +44,11 @@ def __init__(self, *, bucket_name: str, redis_threshold: int) -> None:
 
         self.redis_threshold = redis_threshold
         self.redis_storage = ImplicitRedisStorage()
+        self.storable_kinds = ("result", "payload")
```

Comment:
Looks like a vestige of a past implementation?

Reply:
Removed.